src/main/kotlin/no/nav/familie/ef/iverksett/infrastruktur/IverksettController.kt

```diff
@@ -17,8 +18,8 @@ import org.springframework.web.bind.annotation.RestController
 class IverksettController(val iverksettService: IverksettService) {
 
     @PostMapping(path = ["/iverksett"])
-    fun iverksett(@RequestBody vedtakJSON: IverksettJson): ResponseEntity<String> {
-        return ResponseEntity("Iverksett", HttpStatus.OK)
+    fun iverksettdummy(@RequestBody iverksettJson: IverksettJson): ResponseEntity<IverksettJson> {
```

Comment:
Controllers i Spring, og jersey for så vitt også, håndterer wrapping av entiteter slik att det er ikke nödvendig.
Dvs det er tilstrekkelig å kun returnere `IverksettJson` her